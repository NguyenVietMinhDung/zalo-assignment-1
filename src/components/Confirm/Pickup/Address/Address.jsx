import React from 'react';
import { Row, Col } from 'react-grid-system';
import './Address.scss';
import pickupOrder from '../../../../assets/images/PICKUP_ORDER.png';

const Address = () => (
  <Row className="address">
    <Col xs={3}>
      <img src={pickupOrder} alt="" />
    </Col>
    <Col xs={9}>
      <div className="address__name">
        <div>Highlands Coffee - VNG campus</div>
        <div>Thay đổi</div>
      </div>
      <div className="address__description">
        Z06 Đường số 13, KCX Tân Thuận, P.Tân Thuận Đông, Quận 7
      </div>
    </Col>
  </Row>
);

export default Address;
