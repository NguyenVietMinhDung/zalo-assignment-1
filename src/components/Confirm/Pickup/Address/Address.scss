@import '../../../../styles/variables';

.address {
	padding-top: 15px;
	padding-bottom: 15px;

	border-bottom: 1px solid $white-smoke;

	img {
		width: 100%;

		border-radius: 10px;
		background-color: $white-smoke;
	}

	&__name {
		display: flex;

		margin-bottom: 5px;

		div:first-child {
			flex: 1;

			font-weight: bold;
		}

		div:last-child {
			cursor: pointer;

			color: $light-sky-blue;

			font-size: 12px;
			font-weight: 500;
		}
	}

	&__description {
		opacity: 0.5;

		font-size: 12px;
		font-weight: 500;
	}
}
