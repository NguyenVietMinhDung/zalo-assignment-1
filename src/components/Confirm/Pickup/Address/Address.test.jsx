import React from 'react';
import { shallow } from 'enzyme';
import Address from './Address';

describe('<Address />', () => {
  test('renders correctly', () => {
    const component = shallow(<Address />);
    expect(component).toMatchSnapshot();
  });
});
