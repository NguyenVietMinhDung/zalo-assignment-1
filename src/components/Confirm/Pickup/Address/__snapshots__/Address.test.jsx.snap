// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Address /> renders correctly 1`] = `
<Row
  align="normal"
  className="address"
  component="div"
  debug={false}
  gutterWidth={null}
  justify="start"
  nogutter={false}
  nowrap={false}
  style={Object {}}
>
  <Col
    component="div"
    debug={false}
    lg={null}
    md={null}
    offset={Object {}}
    pull={Object {}}
    push={Object {}}
    sm={null}
    style={Object {}}
    width={null}
    xl={null}
    xs={3}
    xxl={null}
  >
    <img
      alt=""
      src="test-file-stub"
    />
  </Col>
  <Col
    component="div"
    debug={false}
    lg={null}
    md={null}
    offset={Object {}}
    pull={Object {}}
    push={Object {}}
    sm={null}
    style={Object {}}
    width={null}
    xl={null}
    xs={9}
    xxl={null}
  >
    <div
      className="address__name"
    >
      <div>
        Highlands Coffee - VNG campus
      </div>
      <div>
        Thay đổi
      </div>
    </div>
    <div
      className="address__description"
    >
      Z06 Đường số 13, KCX Tân Thuận, P.Tân Thuận Đông, Quận 7
    </div>
  </Col>
</Row>
`;
